Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/EncryptFileViewModel.cs

```diff
@@ -103,6 +114,45 @@ protected override void InitializeModel()
             ContinueOnError.OrderIndex = propertyOrderIndex++;
             ContinueOnError.Widget = new DefaultWidget { Type = ViewModelWidgetType.NullableBoolean };
             ContinueOnError.Value = false;
+
+            MenuActionsBuilder<KeyInputMode>.WithValueProperty(KeyInputModeSwitch)
+                .AddMenuProperty(Key, KeyInputMode.Key)
+                .AddMenuProperty(KeySecureString, KeyInputMode.SecureKey)
+                .BuildAndInsertMenuActions();
+        }
+
+        /// <inheritdoc/>
+        protected override void InitializeRules()
+        {
+            base.InitializeRules();
+            Rule(nameof(KeyInputModeSwitch), KeyInputModeChanged_Action);
+        }
+
+        /// <inheritdoc/>
+        protected override void ManualRegisterDependencies()
+        {
+            base.ManualRegisterDependencies();
+            RegisterDependency(KeyInputModeSwitch, nameof(KeyInputModeSwitch.Value), nameof(KeyInputModeSwitch));
+        }
+
+        /// <summary>
+        /// Files input Mode has changed. Set controls visibility based on selection
```

Comment:
what relevance is this Files input mode? Seems to be a copy/paste issue (check all activities)

Reply:
Changed it Key